app/pubspec.yaml

```
@@ -144,7 +143,10 @@ dependencies:
 
   shake_detector:
     path: ../packages/shake_detector
-  open_filex: ^4.5.0
+#  open_filex: ^4.6.0
+  open_filex:
+    git:
+     url: https://github.com/mufassalhussain/open_filex
```

should reference the specific version though:
```suggestion
     url: https://github.com/mufassalhussain/open_filex
     rev: 7ec56421afc37bb2cc3cb751b9775999cd2d9a61
```

---

Used specific commit of `open_filex` as suggested in below commit.

https://github.com/acterglobal/a3/pull/2648/commits/affe2f2d497b93d1cf8c2c86adb56926e91e8cc3